api_tests/preprints/views/test_preprint_detail.py

```diff
@@ -905,9 +921,9 @@ def test_preprint_is_public_detail(self, app, admin, write_contrib, non_contrib,
         res = app.get(private_url, auth=admin.auth)
         assert res.json['data']['id'] == private_preprint._id
 
-    #   test_private_invisible_to_write_contribs
+    #   test_private_visible_to_write_contribs
         res = app.get(private_url, auth=write_contrib.auth, expect_errors=True)
```

Comment:
This has `expect_errors=True` but the status is 200. How does this pass?

Reply:
Oh whoops. I don't know how that passes.